### PR TITLE
Create README.md to represent the project and author day 1 summary

### PR DESCRIPTION
**Pull request by cnrdgrgry**
# PetSynth
## AI Assisted Virtual Pet Generation
`Group Participators: Conrad, Nico and Susannah`
#### Brief:
Create a website for your new company.
Your company is at the forefront of AI, and has a new service that will amaze the world.
Your job is to put the website together using React.

<details>
<summary>Day 1 - Monday</summary>

 - Created group **Slack** channel
 - Set up a project **Trello** workspace.
 - Brainstormed and came up with business concept: **PetSynth**, *AI assisted virtual pet generation*.
 - Created group project repository here on GitHub.
 - Set up a `prettier.rc` file in project root directory.
 - Utilized an existing *Vite* app and re-purposed by amending Router, Routes, file structure and associated *Nav* components in readiness for new development.
 - Created some core *Redux* code including *`initialState` and *Async Thunk* set up. We came to the decision it was a good idea to get this in fairly early, even though tricky if it requires amendment later.
 - [Deployed to Netlify](https://petsynth.netlify.app/)
 - Added some initial global css styles, primarily `border-box`, `font` and `color` variables

A little communication took place in **Slack**, mostly as a place to share the odd code snippet or image, otherwise we spent most of the day co-coding via **Google Meet** video conference, taking it in turns to write code under the "assistive direction" of the other team members
</details>

<details>
<summary>Day 2 - Tuesday</summary>
</details>

<details>
<summary>Day 3 - Wednesday</summary>
</details>

<details>
<summary>Day 4 - Thursday</summary>
</details>

<details>
<summary>Day 5 - Friday</summary>
</details>

### CODE REVIEW

**Review by sfbennett — COMMENTED** [review]
Looks good! Think we should add here that we are taking a mobile-first approach as well?

**Review by sfbennett — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,10 +1,42 @@
-# React + Vite
+# PetSynth
+## AI Assisted Virtual Pet Generation
+`Group Participators: Conrad, Nico and Susannah`
```
Comment (hedgehog125) [inline README.md:3]: Is this a code block just to make it stand out? Should be Group Participators bit be separate?

Comment (cnrdgrgry) [reply README.md:3]: just to make it stand out - you have the visual example at the top

**Review by hedgehog125 — APPROVED** [review]
Happy to merge if we want to put that on the backlog. I didn't notice anything else, although I didn't spend too long reading it

**Pull request merged**